docs/infra/infra-labels.md

```diff
@@ -0,0 +1,8 @@
+
+# Infra labels
+
+The following labels have special meaning for dotnet/runtime infrastructure, usually to indicate the health of testing and building the repository.
+
+- blocking-outerloop – Blocking any outerloop build, meaning a build not normally run on PRs that is queued on a time schedule.
+- blocking-clean-ci – “Blocking our core PR or CI builds”
+- blocking-clean-ci-optional – “Blocking optional legs in PRs”
```

Comment:
"CI" can (formally) mean "builds that happen after a change is merged", which makes the description "Blocking optional legs in PRs" confusing. Also, IMO, we shouldn't use the word "legs". And what is an "optional" leg anyway? Typically, this would be an "outerloop" pipeline, right? So when would you use this instead of "blocking-outerloop"?

Reply:
We've been trying to use the words "PR validation" and "rolling builds" since CI was so long used to refer to both or one or the other it's essentially burned...

"blocking-validation" ? "broke-validation-recently" ?